guide-us-english.tex

```diff
@@ -347,11 +347,16 @@ \subsection{Strategic Vision}
 
 \subsection{Backlog Prioritization}
 
+Proper Prioritization Prevents Poor Performance.
+Prioritizing the backlog can reveal activities that have negative value, no value or little value.
+Identifying them and eliminating them provides focus and prevents waste.
+Therefore, in Scrum@Scale, a crucial component of the PO cycle is Backlog Prioritization.
+
```

Comment:
Same as previous remark: appreciate the need for an intro, just don't like this one. Feels like a duplication of the list under it.